pkg/publish.go

```diff
@@ -250,3 +260,83 @@ func getBytesSha256(b []byte) string {
 	h.Write(b)
 	return hex.EncodeToString(h.Sum(nil))
 }
+
```

Comment:
What do you think about moving all this to a `publish_links.go`? And maybe move the image functions to `publish_images.go`?

Reply:
- [ ] move linking and image code to separate files